src/lucky/tags/specialty_tags.cr

```diff
@@ -37,8 +37,9 @@ module Lucky::SpecialtyTags
   # See the [MDN's documentation on the meta
   # tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta) for
   # details.
```

Comment:
Can you update these docs to point to https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag ? That way it's easier to see what options can be passed in to this.

Reply:
Great idea!